python/ray/dag/tests/experimental/test_dag_visualization.py

```diff
@@ -8,6 +8,14 @@
 import pytest
 
 
+def cleanup_files(filename: str):
```

Comment:
This can be done as a pytest fixture, but can be done later. We can first merge the PR.

Reply:
Let me add it now. 1 second!